src/containers/access/src/AttachmentItem.js

```diff
@@ -79,14 +106,32 @@ const AttachmentItem = ({
     onDelete(fileId);
   };
 
+  const handleEditTag = () => {
+    setEditing(!editing);
+  };
+
+  const onTagChange = (entityKeyId, value = '') => {
+    dispatch(updateAttachmentTag({ entityKeyId, tag: value }));
+    setEditing(false);
+  };
+
   return (
     <ListItem divider={divider}>
       <ListItemAvatar>
         {imagePreview}
       </ListItemAvatar>
-      <ListItemText primary={name} secondary={dateStr} />
-
+      <ItemTextWrapper paddingRight="96px">
+        <ListItemText
+            primary={name}
+            secondary={!editing ? <SecondaryText date={dateStr} tag={tag} /> : null} />
```

Comment:
I prefer `editing` over `!editing`, easy to miss that `!`

```suggestion
            secondary={editing ? null : <SecondaryText date={dateStr} tag={tag} />} />
```